### ISSUE TITLE
Pytorch lightning callback to freeze backbone

### ISSUE THREAD

**Issue by fstroth** [airctic/icevision#966]
## Pytorch lightning callback to freeze backbone

**Describe the solution you'd like**
Write callback the takes an int for the number of epochs to freeze the backbone for.

Pls assign to me.

**Comment by FraPochetti** [comment]
Do you mean something like [this](https://github.com/FraPochetti/KagglePlaygrounds/blob/fe4edc64c6d48626636d5efec7439f0eba7baf0f/deepfake/code/utils.py#L184) one?

**Comment by fstroth** [comment]
Yes. For a first implementation, I would like to mimic the interface of FastAI where we freeze the backbone for n epochs and then train both. Your version seems to use a gradual approach, which could be one of the next steps.
